packages/techdocs-cli/src/commands/publish/publish.ts

```diff
@@ -57,7 +57,37 @@ export default async function publish(cmd: Command) {
         ...(cmd.awsEndpoint && { endpoint: cmd.awsEndpoint })
       }
     };
-  } else {
+  } else if ("openStackSwift" === cmd.publisherType) {
+    if (!cmd.osUsername) {
+      logger.error(`openStackSwift requires --osUsername to be specified`);
+      throw new Error();
+    } else if (!cmd.osPassword) {
+      logger.error(`openStackSwift requires --osPassword to be specified`);
+      throw new Error();
+    } else if (!cmd.osAuthUrl) {
+      logger.error(`openStackSwift requires --osAuthUrl to be specified`);
+      throw new Error();
+    } else if (!cmd.osRegion) {
```

Comment:
Maybe another way to avoid to repeat logger.error + throw new Error(); each time.
Just set a var and throw new Error if this var is present.

Use parameter too for only the part is changing in your message and store the rest in const

Reply:
I updated the code, is it better ?

Reply:
yes it's look good